app/javascript/App/components/NewsfeedPage/NewsList.js

```diff
@@ -13,7 +13,7 @@ const NewsItemList = (props) => {
         {newsItems
           .reverse()
           .map((newsItem) => (
-            <NewsItemListItemAnimated
+            <NewsListItemAnimated
```

Comment:
nice one! i can't say i will miss you, Mr. `NewsItemListItem` 😢

Reply:
Haha <3